.github/workflows/ci.yml

```diff
@@ -37,20 +37,34 @@ jobs:
       if: runner.os == 'Linux'
       run: |
         sudo apt-get update
-        sudo apt-get install -y pkg-config libelf-dev libboost-dev
-        cmake -B ${{github.workspace}}/build/Release -DAIEBU_FULL=OFF -DCMAKE_BUILD_TYPE=Release 
+        sudo apt-get install -y pkg-config libelf-dev libboost-dev pylint elfutils
+        python -m pip install --upgrade pip
+        pip install pyyaml jinja2 Markdown pylint colorama gitPython wrapt
+        cmake -B ${{github.workspace}}/build/Release -DAIEBU_FULL=ON -DCMAKE_BUILD_TYPE=Release
 
     - name: Install Dependencies (Windows) and configure CMake
       if: runner.os == 'Windows'
       run: |
         vcpkg integrate install
         vcpkg install boost-format boost-uuid boost-interprocess boost-algorithm boost-property-tree boost-headers --triplet=x64-windows
-        cmake -B ${{github.workspace}}/build/Release -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake -DAIEBU_FULL=OFF -DCMAKE_BUILD_TYPE=Release 
+        python3 -m pip install --upgrade pip
+        python3 -m pip install pyyaml jinja2 Markdown pylint colorama gitPython wrapt
+        cmake -B ${{github.workspace}}/build/Release -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake -DAIEBU_FULL=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/build/Release
 
     - name: Build
       run: |
         cmake --build ${{github.workspace}}/build/Release --config Release
 
+    - name: Install (Linux)
+      if: runner.os == 'Linux'
+      run: |
+        make -C ${{github.workspace}}/build/Release DESTDIR=${{github.workspace}}/build/Release install
```

Comment:
Let use `cmake --build`, so you want to add -DCMAKE_INSTALL_PREFIX to Linux as you did for Windows.
The lets move the cmake install command to line 57 under Build (rename to Build and Install)